src/lib/header-bidding/prebid-types.ts

```diff
@@ -27,6 +27,7 @@ export type HeaderBiddingSizeKey =
 export type HeaderBiddingSlot = {
 	key: HeaderBiddingSizeKey;
 	sizes: HeaderBiddingSize[];
+	name: string;
```

Comment:
is the name needed, can't `key` be used for the gpid?

Reply:
thanks 👍 yes indeed we can!